### ISSUE TITLE
Bernoulli distribution has a bug in it

### ISSUE THREAD

**Issue by shell769324** [NM512/dreamerv3-torch#29]
https://github.com/NM512/dreamerv3-torch/blob/8fa2274cfc0d78b15aeb28a424ab35742fb750e5/networks.py#L715

According to [pytorch doc]( https://pytorch.org/docs/stable/generated/torch.bernoulli.html), Bernoulli should directly take in the mean as the first argument instead of using `logits=...`

The result of doing `logits=...` is the predicted value gets ignored, and the default `0.5` will be used
